index.html

```diff
@@ -844,7 +848,20 @@ <h4>publicKeyJwk</h4>
 
       <section>
         <h4>publicKeyBase58</h4>
+        <span class="label-deprecated">Deprecated</span>
 
+        <p class="advisement">
+This property is deprecated in favor of <code>publicKeyMultibase</code> or
+<code>publicKeyJwk</code>. It's generally expected that this term will still be
+used in older suites and therefore needs be supported for legacy compatibility,
+but is expected to not be used for newly defined suites.
+        </p>
+
+        <p class="note">
```

Comment:
Since this same note is included for every verification method property would it make sense to elevate this to a note of the sub-section rather than copying and pasting it for every entry in this sub-section?

Reply:
sounds good, open an issue, don't make this PR any bigger :)

Reply:
Making this change would make this PR smaller not bigger since it would be removing repetition in place of a single note at the top of the section. Also, this PR is relatively small at 90 additions, most of which are coming from a copy and pasted note.